source/includes/examples/cli-example-backup-compliance-policy-schedule.rst

```diff
@@ -3,7 +3,7 @@
 
    atlas backups compliancePolicy policies scheduled create \ 
      --projectId 67212db237c5766221eb6ad9 \
-     --frequencyInterval 2 \
```

Comment:
Question: Do we need to update these for Terraform examples as well?

Reply:
No, because we have different examples for Terraform.